research-2023/effort-mapping-model/2023-04-28.001.md

```diff
@@ -0,0 +1,321 @@
+---
+marp: true
+title: "Define: Mapping Model Revisions"
+author: Chris Compton
+class: invert workshop
+footer: "![height:50px](https://www.nist.gov/libraries/nist-component-library/dist/img/logo/NIST-Logo-Brand-White.svg)"
+header: "OSCAL DEFINE: Mapping Model Revisions"
+paginate: true
+---
+
+
+# Define: Mapping Model Revisions
+
+---
+## Problem
+
+Relates to: OSCAL control mapping model support for identifying controls in a catalog that are not mapped, and other relational concerns when mapping a source and target. [^Issue-1332][^Issue-1333]
+
+All issues related to mapping requirements and feedback need to be consolidated and synthesized. Findings were produced at the end of 2022, but not shared. These need to be summarized as a part of the effort.
+
+---
+
+| Element                        | Response                                                                      |
+| ------------------------------ | ----------------------------------------------------------------------------- |
+| Current Spiral Sequence Number | [2023-04-28.001.md](2023-04-28.001.md)                                        |
+| GitHub Project Link            | https://github.com/usnistgov/OSCAL/                                           |
+| GitHub Issue #                 | 1332[^Issue-1332] 1333[^Issue-1333]                               |
+| Author(s)                      | Chris Compton                                                                 |
+| Impact                         |                                                                               |
+| Criticality                    | Significant                                                                   |
+| Scope                          | Project, Public Capabilities                                                  |
+| Audience                       | Control Framework Mapping Experts, Framework Organizations, GRC Practitioners |
+
+---
+
+## Constraints
+
+- Mapping model examples and/or concepts will not be addressed until a future spiral
+
+
+## Requirements
+
+- Compile recent research prior to the initiation of OSCAL DEFINE
+
+## Approach                    
+
+- Literature and Issue Review
+
+## Participants
+
+- No other participants identified.
+
+---
+
+# Discover
+---
+
+## Capability: Gap Specification
+
+Feedback was received to consider a tag that would support identifying gaps in a mapping.
```

Comment:
My advice to Chris, during the meeting today, was to not focus on a 'tailored-relation' field. The ask from the community was to be able to note a new relation if a profile sets parameters that help align the controls or if an implementer makes implementation decisions that facilitate better alignments, because those will not be mandatory for all users/cases and we can add them as optional after performing more research and receiving feedback from the community.

Reply:
OK, thanks to both of you, you helped me understand that it appears we are all on the same page on this around the control or statement-level granularity based upon the feedback provided by some community members.